docs/user-guide/admin-cli.md

```diff
@@ -454,25 +454,33 @@ Datastore   Allow_create  Max_volume_size  Total_size
 datastore1  True          Unset            Unset
 datastore2  True          500.00MB         1.00GB
 ```
-For _VM_DS and _ALL_DS special DS names, --volume-totalzie has to be "Unset".
+For _VM_DS and _ALL_DS special DS names, --volume-totalzie can also be set.
+
+When "--volume-totalsize" is set for "_VM_DS", it means the total volume size on datastore where VM lives cannot exceed the value specified by "--volume-totalsize". For example, if "--volume-totalsize" is set to "1GB" for "_VM_DS" and VM lives in "datastore2", where the total volume size is already 800MB and user tries to create another volume with 500MB will not be allowed. However, if the VM is moved to "datastore3", where the total volume size is 500MB and user tries to create another volume with 500MB will be allowed.
+
 ```
 [root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=vmgroup1 --datastore=_VM_DS  --volume-maxsize=500MB --volume-totalsize=1GB --allow-create
-Canont set volume-totalsize for _VM_DS
-[root@localhost:~]
-[root@localhost:~]
-[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=vmgroup1 --datastore=_ALL_DS  --volume-maxsize=500MB --volume-totalsize=1GB --allow-create
-Canont set volume-totalsize for _VM_DS
-[root@localhost:~]
-[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=vmgroup1 --datastore=_ALL_DS  --volume-maxsize=500MB  --allow-create
 vmgroup access add succeeded
-[root@localhost:~]
+
 [root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access ls --name=vmgroup1
-Datastore   Allow_create  Max_volume_size  Total_size
-----------  ------------  ---------------  ----------
-datastore1  True          Unset            Unset
-_ALL_DS    True          500.00MB         Unset
+Datastore     Allow_create  Max_volume_size  Total_size
+------------  ------------  ---------------  ----------
+datastore1    True          Unset            Unset
+_VM_DS        True          500.00MB         1.00GB
+```
+
+When "--volume-totalsize" is set for "_ALL_DS", it means the total volume size on any datastore which has not been given access to explicitly cannot exceed the value specified by "--volume-totalsize". In the following example, "vmgroup1" has been given access to "datastore1" and "_ALL_DS". The "Total_size" for "datastore1" is "Unset", which means no limit. The "Total_size" for "_ALL_DS" is "1GB". So for "datastore1", there is no limit on the total volume size. However, for any other datastore, the total volume size on that datastore cannot exceed 1GB.
```

Comment:
Could clarify explicitly whether the volume-totalsize applies across all those datastores that aren't part of the vmgroup or applies per each of those datastores. I'm assuming the former, the latter implies that user can create volume-totalzie worth VMDKs on each datastore thats explicitly not in the vmgroup.